anago

```diff
@@ -622,6 +622,21 @@ prepare_tree () {
   #            "Generating docs for ${RELEASE_VERSION[$label]} on $branch."
   #fi
 
+  # Clear all CHANGELOG-N.NN.md files on the branch.
+  # This replaces the above commented out change to the branch.
+  # Some unique change is required on the branch to allow it to be git
+  # describe'd as something other than the current tag on master
+  # If/when the generate-docs.sh issue is sorted out, this will become
+  # optional, though probably still useful anyway.
+  if [[ "$PARENT_BRANCH" == "master" && "$branch" != "master" ]]; then
+    logecho -n "Remove any previous CHANGELOG-*.md files: "
+    logrun -s git rm -f CHANGELOG-*.md || return 1
```

Comment:
It looks like the CHANGELOG for the new release will already exist on master at this point, since it gets populated for alphas. Should we avoid deleting the one that matches the version in the branch name?

Reply:
Yes, that had got lost of the recent shuffle. Thanks. PTAL.